feature-policy/interfaces.any.js

```diff
@@ -0,0 +1,28 @@
+// META: script=/resources/WebIDLParser.js
+// META: script=/resources/idlharness.js
+
+"use strict";
+
+if (self.importScripts) {
```

Comment:
This bits ought to not be needed, sent https://github.com/w3c/web-platform-tests/pull/10033 to confirm.